src/operator/src/bulk_insert.rs

```diff
@@ -83,84 +90,88 @@ impl Inserter {
         let mut mask_per_datanode = HashMap::with_capacity(region_masks.len());
         for (region_number, mask) in region_masks {
             let region_id = RegionId::new(table_id, region_number);
+            metrics::BULK_REQUEST_ROWS
+                .with_label_values(&["rows_per_region"])
+                .observe(mask.true_count() as f64);
```

Comment:
Is it(compute true_count) an expensive operation?

Reply:
Maybe we can observe this after we filter the batch. Then we can use the num_rows of the batch.

Reply:
Can we also include `region_id` in the labels?

Reply:
> Can we also include `region_id` in the labels?

That will bring too many buckets